src/main/java/codeu/model/data/Conversation.java

```diff
@@ -42,12 +44,34 @@ public Conversation(UUID id, UUID owner, String title, Instant creation) {
     this.title = title;
   }
 
+  /**
+   * Constructs a new Conversation.
+   *
+   * @param id the ID of this Conversation
+   * @param owner the ID of the conversation who created this Conversation
+   * @param title the title of this Conversation
+   * @param creation the creation time of this Conversation
+   * @param messageCount the message count of this Conversation
+   */
+  public Conversation(UUID id, UUID owner, String title, Instant creation, long messageCount) {
+    this.id = id;
+    this.owner = owner;
+    this.creation = creation;
+    this.title = title;
+    this.messageCount = messageCount;
+  }
+
+  /** Increases the count of messages fo this conversation by one*/
+  public void increaseMessageCount() {
+    messageCount++;
+  }
+
   /** Returns the ID of this Conversation. */
   public UUID getId() {
     return id;
   }
 
-  /** Returns the ID of the User who created this Conversation. */
+  /** Returns the ID of the conversation who created this Conversation. */
```

Comment:
Lets change all of these back to "User". Each conversation is associated with the user who created it.